absolventenfeier/einladung.tex

```diff
@@ -41,9 +41,9 @@
 
 Liebe Absolventin oder lieber Absolvent,
 
-zu deinem in den Jahren 2019 – 2022 an der TU Dortmund erworbenen Abschluss möchten
+zu deinem in den Jahren 2019 – 2022 an der TU Dortmund erworbenen Master-, Diplom- oder Promotionsabschluss möchten
```

Comment:
Is this necessary if we only invite the relevant people?

Reply:
no